exp-models/addon/models/account.js

```diff
@@ -2,14 +2,34 @@
 Manage data about one or more documents in the accounts collection
  */
 
+import moment from 'moment';
```

Comment:
Is this line necessary, given that we use `app.import` in `ember-cli-build.js`?

https://guides.emberjs.com/v2.0.0/addons-and-dependencies/managing-dependencies/#toc_globals-provided-by-javascript-assets